designing-reports/report-designer-tools/web-report-designer/web-report-designer-initialization.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Web Report Designer Initialization
+page_title: Web Report Designer Initialization
+description: Web Report Designer Initialization
+slug: telerikreporting/report-designer-tools/web-report-designer/web-report-designer-initialization
+tags: report,webreportdesigner,initialization
+published: True
+position: 7
+---
+<style>
+table th:first-of-type {
+    width: 28%;
+}
+table th:nth-of-type(2) {
+    width: 72%;
+}
+</style>
+
+# Web Report Designer Initialization
+
+The Telerik Web Report Designer is a jQuery plugin - __jQuery.fn.telerik_WebReportDesigner(options)__. Below is a list of all options available during initialization. 
+
+## Options
+
+
+| Parameter | Description |
+| ------ | ------ |
+| __id__ | *string*, *optional*; Sets the unique identifier of the WebReportDesigner instance;|
+| __serviceUrl__ | *string*, *required*; Sets the URL of the service which will provide for the web report designer client (relative or absolute);|
```

Comment:
Consider something like:
'Sets the URL (relative or absolute) which will provide the service for the web report designer client;'
or like in the Html5 viewer options:
'Sets the address of the Designer REST Service;'